### PR TITLE
test: fix tests

### PR DESCRIPTION
**Pull request by mxschmitt**
The Playwright Test version inside the repository is pinned - this means like we installed the dependencies of the pinned version (GitHub workflow yml).

Since Playwright evolves -> more dependencies get added over time. This diverges with the actual Playwright version inside the tests -> latest. This change calls install-deps inside the tests too. An alternative approach would be #12 but this seems nicer.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -6,11 +6,12 @@ on:
     branches: [ main ]
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
         node-version: [12.x, 14.x, 16.x]
+        os: [ubuntu-latest, windows-latest, macos-latest]
```
Comment (mxschmitt) [inline .github/workflows/ci.yml:14]: drive-by: ensure it works on all the operating systems.

File: playwright.config.ts [hunk 2]
```diff
@@ -14,11 +14,11 @@
  * limitations under the License.
  */
 import { PlaywrightTestConfig } from '@playwright/test';
-import path from 'path';
 
 const config: PlaywrightTestConfig = {
   timeout: 120 * 1000,
   testDir: './tests',
+  reporter: [['list'], ['github']],
```
Comment (mxschmitt) [inline playwright.config.ts:21]: drive-by: dot reporter does us not give us fast feedback inside a CI run since it splits only after 80 dots.
drive-by: dog GitHub reporter.

**Review by dgozman — APPROVED** [review]
